utils/snap.go

```diff
@@ -124,14 +124,15 @@ func snapJournalCommand(start time.Time, name string) string {
 		name)
 }
 
-func SnapDumpLogs(t *testing.T, start time.Time, name string) {
-	filename := name + ".log" // used in action.yml
+func SnapDumpLogs(t *testing.T, start time.Time, snapName string) {
+	logFileName := GetLogFileName(t, snapName)
+
 	ExecVerbose(t, fmt.Sprintf("(%s) > %s",
-		snapJournalCommand(start, name),
-		filename))
+		snapJournalCommand(start, snapName),
+		logFileName))
 
 	wd, _ := os.Getwd()
-	fmt.Printf("Wrote snap logs to %s/%s\n", wd, filename)
+	fmt.Printf("Wrote snap logs to %s/%s\n", wd, logFileName)
```

Comment:
Should this be changed to `t.Log` or become conditional based on the value of `t`?